pkg/controllers/capr/bootstrap/controller.go

```diff
@@ -438,16 +442,15 @@ func (h *handler) reconcileMachinePreTerminateAnnotation(bootstrap *rkev1.RKEBoo
 		return bootstrap, nil
 	}
 
-	clusterName := bootstrap.Labels[capi.ClusterLabelName]
```

Comment:
Is there any reason we are using the spec value for clustername that we set as opposed to the CAPI cluster label value?

Reply:
I don't think we need to remove the field, and if we do, it technically needs us to bump our API version.

We also use the field for indexing IIRC